File: app/models/fridge.rb
class Fridge < ActiveRecord::Base
  belongs_to :user
  has_many :fridgecategories
  has_many :groceries, through: :fridgecategories

  validates :name, presence: true
  validates :user_id, presence: true
  validates :user_id, numericality: { only_integer: true }
end

File: react/src/components/Fridge.js
import React from 'react';
import FridgeCategory from './FridgeCategory';

const Fridge = props => {
  let fridgeCategories = props.groceries.map(fridgeCategory => {
    return (
      <FridgeCategory
        key={id}
        name={name}
      />
    );
  });

  return (
    <ul>
      <li>{fridgeCategories}</li>
    </ul>
  );
};

export default FridgeCategory;
